### ISSUE TITLE
Why does the loss value decrease so quickly, and why is the accuracy (acc) so high ?

### ISSUE THREAD

**Issue by lmj282371** [MilleXi/stock_trading#4]
Why does the loss value decrease so quickly, and why is the accuracy (acc) so high ? Is this normal? After training for ten epochs, the loss basically drops to between 0.01 and 0, and the accuracy also reaches 99%. Is this normal? Could it be overfitting?

**Comment by MilleXi** [comment]
我检查过代码，应该没有数据泄露，也应该不至于涉及过拟合的问题，你可以尝试使用模型进行推理，看看验证与测试的情况
